cannon/mipsevm/tests/evm_multithreaded_test.go

```diff
@@ -327,9 +328,9 @@ func TestEVM_SysClone_Successful(t *testing.T) {
 			expectedNewThread.PC = state.GetCpu().NextPC
 			expectedNewThread.NextPC = state.GetCpu().NextPC + 4
 			expectedNewThread.ThreadId = 1
-			expectedNewThread.Registers[2] = 0
-			expectedNewThread.Registers[7] = 0
-			expectedNewThread.Registers[29] = stackPtr
+			expectedNewThread.Registers[register.RegSyscallNum] = 0
```

Comment:
Since 0 is returned in the child process:
```suggestion
			expectedNewThread.Registers[register.RegSyscallRet1] = 0
```